packages/noise/package.json

```diff
@@ -20,7 +20,7 @@
     "prepack:cjs": "mv package.json package.json.bak && jq \".main = \\\"dist/cjs/index.cjs\\\"\" package.json.bak > package.json.tmp && jq \".module = \\\"dist/index.js\\\"\" package.json.tmp > package.json.tmp2 && jq \"del(\".type\")\" package.json.tmp2 > package.json",
     "prepack": "mv package.json package.json.bak && jq \".main = \\\"dist/index.js\\\"\" package.json.bak > package.json",
     "postpack": "(mv package.json.bak package.json || true) && (rm package.json.tmp || true) && (rm package.json.tmp2 || true)",
-    "build": "rm -rf && yarn asbuild:release && yarn prepack:cjs && yarn parcel build && yarn postpack",
```

Comment:
I don't think this `rm -rf` is doing anything without a file arg?

Reply:
looks like it used to remove `dist` https://github.com/latticexyz/mud/commit/fb61b0326a84fe1e11dafeae240a0c1fe6bd0f2f